jsonnet/telemeter/server/kubernetes.libsonnet

```diff
@@ -126,7 +123,6 @@ local clusterPort = 8082;
 
       secret.new(secretName, {
         'rhd.url': std.base64($._config.telemeterServer.rhdURL),
-        'rhd.username': std.base64($._config.telemeterServer.rhdUsername),
         'rhd.password': std.base64($._config.telemeterServer.rhdPassword),
```

Comment:
is "password" still appropriate here? would "secret" not be more appropriate?

Reply:
yes, we can make that change. then the args will have `--password=$SECRET` because https://github.com/openshift/telemeter/pull/208 recycles the same args

Reply:
Ack, let's do all of that in one go